### PR TITLE
www-apps/kibana-bin: version bump to 4.6.0

### PR DESCRIPTION
**Pull request by hydrapolic**
Package-Manager: portage-2.3.0

### CODE REVIEW

File: www-apps/kibana-bin/kibana-bin-4.6.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+QA_PREBUILT="opt/kibana/node/bin/node"
+
+pkg_setup() {
+	enewgroup ${MY_PN}
+	enewuser ${MY_PN} -1 -1 /opt/${MY_PN} ${MY_PN}
+}
+
+src_unpack() {
+	if use amd64; then
+	  S="${WORKDIR}/${MY_P}-linux-x86_64"
```
Comment (SoapGentoo) [inline www-apps/kibana-bin/kibana-bin-4.6.0.ebuild]: not indented properly

File: www-apps/kibana-bin/kibana-bin-4.6.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	elif use x86; then
+	  S="${WORKDIR}/${MY_P}-linux-x86"
```
Comment (SoapGentoo) [inline www-apps/kibana-bin/kibana-bin-4.6.0.ebuild]: not indented properly

File: www-apps/kibana-bin/kibana-bin-4.6.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	doins config/*
+	rm -rf config || die
+
+	insinto /etc/logrotate.d
+	newins "${FILESDIR}"/${MY_PN}.logrotate ${MY_PN}
+
+	newconfd "${FILESDIR}"/${MY_PN}.confd ${MY_PN}
+	newinitd "${FILESDIR}"/${MY_PN}.initd-r3 ${MY_PN}
+
+	mv * "${D}"/opt/${MY_PN} || die
```
Comment (SoapGentoo) [inline www-apps/kibana-bin/kibana-bin-4.6.0.ebuild]: Change `"${D}"` to `"${ED%/}"`

File: www-apps/kibana-bin/kibana-bin-4.6.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+	# bug 567934
+	pax-mark m "${ED}/opt/${MY_PN}/node/bin/node"
```
Comment (SoapGentoo) [inline www-apps/kibana-bin/kibana-bin-4.6.0.ebuild]: Change `${ED}` to `${ED%/}` to prevent double slashes

**Comment by gktrk** [comment]
@gentoo/proxy-maint

**Comment by hydrapolic** [comment]
Thanks @SoapGentoo  for the comments. As @patricklauer  already bumped to version 2.4.0, we can take the improvements and apply to 2.4.0 that is in the tree already -> https://github.com/gentoo/gentoo/pull/2241.

**Pull request closed without merging**